process.c

```diff
@@ -8669,6 +8644,172 @@ proc_warmup(VALUE _)
  * \Module +Process+ represents a process in the underlying operating system.
  * Its methods support management of the current process and its child processes.
  *
+ * == \Process Creation
+ *
+ * Each of these methods creates a process:
+ *
+ * - Process.exec: Replaces the current process by running a given external command.
+ * - Process.spawn: Creates a child process.
+ * - Kernel#spawn: Executes the given command and returns its pid without waiting for completion.
```

Comment:
I'm not sure why the documentation for `Process.spawn` is different from `Kernel#spawn`. They both call the same C function `rb_f_spawn`.

Reply:
I took these from the two What's Here sections.  If not ok, what should we say here?

Reply:
Maybe we can group them together into a single bullet point? Or maybe be separate bullet points have their descriptions be the same? Maybe we can mention that they are the same method? I'm not sure what the best solution here is.

Reply:
Grouped.